cmd/codegen/generator/client_set.go

```diff
@@ -25,14 +26,14 @@ var (
 )
 
 // InstallCodeGenerators installs client-gen from the kubernetes-incubator/reference-docs repository.
-func InstallCodeGenerators(version string) error {
+func InstallCodeGenerators(version string, gopath string) error {
```

Comment:
might be worth to rename these variables to something like `toolGoPath` to create a bit of a differentiation

Reply:
That would be confusing - what's a "Tool Go Path"? This is just the `GOPATH` to use.